src/main/java/frc/robot/subsystems/gpm/Arm.java

```diff
@@ -186,10 +180,10 @@ public Arm() {
             encoderSim.setDistance(ArmConstants.START_ANGLE_RADS);
 
             // put the display on the SmartDashboard
-            SmartDashboard.putData("ArmSim", wristDisplay);
-            SmartDashboard.putData("arm pid", pid);
+            // SmartDashboard.putData("ArmSim", wristDisplay);
+            // SmartDashboard.putData("arm pid", pid);
         }
-        Timer.delay(1);
+        Timer.delay(2);
 		double cachedAngleRad = getAngleRad(); // don't get the angle five times
```

Comment:
This is poor programming practice. Do not wait for the result; instead read the data later.

Reply:
This is to disable the arm on deploy if it is out of range, but we had to wait so it would get the current angle instead of the angle we turned it off at.